studio/data/organizations/organization-audit-logs-query.ts

```diff
@@ -32,7 +33,7 @@ export type OrganizationAuditLog = {
 }
 
 export type OrganizationAuditLogsResponse = {
-  result: OrganizationAuditLog[]
+  result: AuditLog[]
   retention_period: number
 }
 
```

Comment:
`iso_timestamp_start` and `iso_timestamp_end` are not actually optional. The API sends an error if you don't supply them.